docs/miscellaneous.rst

```diff
@@ -303,6 +303,124 @@ The following is the order of precedence for operators, listed in order of evalu
 
 .. index:: assert, block, coinbase, difficulty, number, block;number, timestamp, block;timestamp, msg, data, gas, sender, value, now, gas price, origin, revert, require, keccak256, ripemd160, sha256, ecrecover, addmod, mulmod, cryptography, this, super, selfdestruct, balance, send
 
+Overflow and underflow rules for operators
+==========================================
+
+Integers in Solidity have a fixed range of possible values, and that range is defined by their bit-size. When an (arithmetic) operation attempts to create a number that is greater than the maximum value it results in an *overflow*. If it attempts to create a number that is less than the minimum value allowed then it is called an *underflow*.
+
+Unsigned integers are restricted to the range ``[0, 2**N - 1]``, where ``N`` is the bitsize.
```

Comment:
You write `bit-size` above and `bitsize` here.

Reply:
bitsize is used everywhere in next push.

Reply:
pushed and checked.